pkg/util/coverage/fake_test_deps.go

```diff
@@ -61,3 +63,27 @@ func (fakeTestDeps) WriteHeapProfile(io.Writer) error {
 func (fakeTestDeps) WriteProfileTo(string, io.Writer, int) error {
 	return nil
 }
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) CheckCorpus([]interface{}, []reflect.Type) error {
+	return nil
+}
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) SetPanicOnExit0(bool) {}
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) CoordinateFuzzing(time.Duration, int64, time.Duration, int64, int, []corpusEntry, []reflect.Type, string, string) error {
+	return nil
+}
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) RunFuzzWorker(func(corpusEntry) error) error {
+	return nil
+}
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) ResetCoverage() {}
+
+//nolint:unused // U1000 see comment above, we know it's unused normally.
+func (fakeTestDeps) SnapshotCoverage() {}
```

Comment:
https://github.com/google/gvisor/blob/0968254ce715c0d8e6457558d55cb405a545ad2f/test/runtimes/runner/lib/lib.go#L212-L241 ??

Reply:
it seems you have to copy `corpusEntry` 

```
// Copied from testing/fuzz.go.
type corpusEntry = struct {
	Parent     string
	Path       string
	Data       []byte
	Values     []interface{}
	Generation int
	IsSeed     bool
}
```